includes/acl/decompression/decompress_data.h

```diff
@@ -215,10 +215,7 @@ namespace acl
 				const bool are_segment_rotations_normalized = is_enum_flag_set(segment_range_reduction, RangeReductionFlags8::Rotations);
 
 				Vector4_32 rotations[num_key_frames];
-
-				bool ignore_segment_range[num_key_frames];
-				for (size_t i = 0; i < num_key_frames; ++i)
-					ignore_segment_range[i] = false;
+				bool ignore_segment_range[num_key_frames] = {};
```

Comment:
bool ignore_segment_range[num_key_frames] = { false };
By leaving {} empty, isn't the memory effectively uninitialized? Clearer to explicitly write false which will be repeated to as many needed values as num_key_frames similar to struct foo = {0};

Reply:
I'm happy to change it, though http://en.cppreference.com/w/c/language/array_initialization has the example

int a[3] = {}; // invalid C but valid C++ way to zero-out a block-scope array

Reply:
Interesting, you can leave it like that in that case.